OpenRA.Mods.Common/Scripting/Global/TriggerGlobal.cs

```diff
@@ -211,8 +211,26 @@ public void OnAllRemovedFromWorld(Actor[] actors, LuaFunction func)
 						return;
 
 					if (!group.Any())
-						using (f)
-							f.Call().Dispose();
+					{
+						// Do not dispose the 'original' function f, we may need it again
+						var temp = (LuaFunction)f.CopyReference();
```

Comment:
Do we really need to copy this and then dispose the copy? Can't we just call the original without disposing?

Reply:
The function does not seem to be reusable after `Call()` ran.